assets/css/woocommerce.scss

```diff
@@ -48,6 +48,10 @@ p.demo_store {
 	clear: both;
 }
 
+option[hidden] {
+	display: none;
```

Comment:
Is this needed because we're adding removing DOM elements or something? I thought `hidden` elements were hidden automatically?

If it's required then I'm a little concerned. Lots of themes of course do not load our CSS. 

If this style isn't present in the theme, options are going to be visible that shouldn't be (?)

If that's the case this probably shouldn't go in a fix release imo, unless we make the styles inline or something.